quickstarts/202-coe-starter-kit/readme.md

```diff
@@ -0,0 +1,187 @@
+# Usage
+
+1. `terraform init`
+1. `terraform plan --var-file=prod.tfvars`
+1. `terraform apply --var-file=prod.tfvars`
+1. `terraform destroy --var-file=prod.tfvars`
+
+## Prerequistes
+
+1. The bash shell script [boostrap.sh](../../bootstrap/README.md) has been run
+
+1. [Install Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+
+1. Power Platform CLI (pac) has been downloaded and installed
+
+1. Cross platform version of [PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell) has been downloaded and installed
+
+1. Azure Command CLI has been downloaded and installed
+
+1. The logged in with api scope. For example to login with device code and specify the required tenant the following command could be used
+
+> [!NOTE]
+> Currently to create Power Platform connections you have to use service principal authentication.
+
+```bash
+az login --service-principal -u 01234567-1111-2222-3333-444455556666 -p abcdef1234354567890 --tenant 01234567-1111-2222-3333-444455556666
+```
+
+
+
+### Linux
+
+Required linux commands as part of GitHUb DevContainer. For example
+
+    - (Debian/Ubuntu): Run `sudo apt-get install jq`
+    - [Install the Azure CLI on Linux](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux)
+
+## Login
+
+Assumes that environment as been logged in with api scope. For example to login with device code and specify the required tenant the following command could be used
+
+```bash
+az login --allow-no-subscriptions --scope api://power-platform_provider_terraform/.default --use-device-code --tenant 01234567-1111-2222-3333-444455556666
+```
+
+## Example tfvars file
+
+```hcl
+release_parameters = {
+  coe_starter_kit_get_latest_release   = true,
+  coe_starter_kit_specific_release_tag = "",
+
+  creator_kit_get_latest_release   = false,
+  creator_kit_specific_release_tag = "CreatorKit-May2024",
+}
+
+environment_parameters = {
+  env_name     = "coe-kit-prod",
+  env_location = "europe",
+}
+
+release_parameters = {
+    coe_starter_kit_get_latest_release = true,
+    coe_starter_kit_specific_release_tag = "",
+    creator_kit_get_latest_release = true,
+    creator_kit_specific_release_tag = ""
+}
+
+core_components_parameters = {
+  admin_admine_mail_preferred_language                        = "en-US",
+  admin_admin_mail                                            = "admin@contoso.com",
+  admin_app_connections_dataflow_id                           = "", //empty for import //what is that?
```

Comment:
should we add comments explaining those parameters? Otherwise we should remove those comments completly